src/modules/project/pages/projectView/views/projectActivityPanel/screens/info/views/ActivityBrief.tsx

```diff
@@ -39,6 +40,7 @@ export const ActivityBrief = (props: StackProps) => {
   const { colors } = useTheme()
 
   const fundersModal = useProjectFundersModal()
```

Comment:
Does the contributorsModal replace the fundersModal right above?

Reply:
No they are two different modals